CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## [2.0.0] - 2022-08-01
+
+### Changed
+- Breaking change: you now need a `version.txt` file at the root of your repository. This file only contains a version number which is used by taskgraph for release-related operations
```

Comment:
I would prefer if we didn't make this mandatory.  Not all repos want/need release-related operations.

Reply:
Discussed offline. Breaking change removed. We don't want to force all projects to define a `version.txt` file. Some projects are meant to be continuous delivered, making version numbers useless.